sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/transform/BeamAggregationTransforms.java

```diff
@@ -223,7 +222,7 @@ public AggregationAccumulator addInput(AggregationAccumulator accumulator, BeamR
       for (int idx = 0; idx < aggregators.size(); ++idx) {
         deltaAcc.accumulatorElements.add(
             aggregators.get(idx).add(accumulator.accumulatorElements.get(idx),
-            sourceFieldExps.get(idx).evaluate(input).getValue()));
+            sourceFieldExps.get(idx).evaluate(input, null).getValue()));
```

Comment:
I assume this is OK because you're not allowed to put XXX_{START,END} operations into an aggregator? Might deserve a comment at least.